config/env/development.example.env

```diff
@@ -12,6 +12,10 @@ GOOGLE_CLIENT_ID=""
 GOOGLE_CLIENT_SECRET=""
 GOOGLE_APP_NAME=""
 
+# Configuration values for Oauth through Facebook
```

Comment:
We will have to setup a way for folks with the existing codebase to update their env files. This can either be done manually by telling folks on Slack/documentation or we can write an automated script for this.

Reply:
Good point. I am gonna see how to make a scrip for this!